acme/acme/standalone_test.py

```diff
@@ -1,6 +1,8 @@
 """Tests for acme.standalone."""
+import mock
```

Comment:
super nit: Since this isn't part of the standard library, it should be moved down with the other 3rd party imports according to our [style](https://www.python.org/dev/peps/pep-0008/#imports) [guides](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting).